webclient/static/js/dz.js

```diff
@@ -217,7 +219,13 @@ window.onload = function () {
                 file.previewElement.lastChild.style.display = "none"
             });
             this.on("addedfile", function (file) {
-                var submitButton = $(".btn-disable-when-running-upload");
+                //hack pro win10
```

Comment:
Vycházíš taky z vlastní zkušenosti nebo z nějakého zdroje?

Reply:
Je to moje zkušenost. Každý operační systém detekuje soubory trochu jinak. Nenašel jsem ani spolehlivou metodu jak detekovat win10 v javascriptu a počítám, že až přestanou být win10 používány, tak tyto řádky smažeme. Testoval jsem to ve win10 i win11, nevím jak se to chová na apple nebo linuxu.